docs/data/horizon/admin-guide/prerequisites.mdx

```diff
@@ -14,19 +14,25 @@ For a basic setup using the [Single Instance Deployment model](./configuring.mdx
 - One for hosting the Horizon service
 - Another for hosting the PostgreSQL server
 
+### Hardware requirements
+
 The minimum hardware specifications to effectively run Horizon are as follows:
 
-### Horizon Compute Instance:
+#### Horizon Compute Instance:
+
+| Node Type | CPU | RAM | Disk | AWS SKU | Google Cloud SKU |
+| --- | --- | --- | --- | --- | --- |
+| Horizon API Service | 4 vCPU | 16 GB | 100 GB SSD >= 1.5K IOPS | [c5d.xlarge] | [n4-standard-4] |
```

Comment:
I realize this wasn't part of change, but, I think this would be good time to just state `3K IOPS` instead as min since `1.5K IOPS` seems to be splitting hairs unnecessarily.